### DISCUSSION TITLE
Retrieve subcollections from firebase wrapper

### DISCUSSION THREAD

**Discussion by Weulass** [supabase/supabase#26632] [Feature Requests]
Is there any way to retrieve all documents associated with a subcollection presents in a collection using the firebase wrapper ? 
I tried without success: 
server firebase_server
  options (
    object 'firestore/users/details'  -- format: 'firestore/[collection_id]'
  );

In this case, details would be a subcollection in users collection
Thanks in advance

**Comment by imor** [comment]
This is not currently possible because the [regex in the code](https://github.com/supabase/wrappers/blob/ce168424f156bf6d64a525b1e3e5c6ee561141f4/wrappers/src/fdw/firebase_fdw/firebase_fdw.rs#L186) matches only `firestore/collection` format while building the API url. But we should be able to update to allow subcollections in the regex. WDYT @burmecia?
